src/oscar/apps/dashboard/catalogue/forms.py

```diff
@@ -24,12 +24,22 @@
                                                   ('RelatedFieldWidgetWrapper',
                                                    'RelatedMultipleFieldWidgetWrapper'))
 
-CategoryForm = movenodeform_factory(
+
+BaseCategoryForm = movenodeform_factory(
     Category,
-    fields=['name', 'description', 'image', 'is_public'],
+    fields=['name', 'slug', 'description', 'image', 'is_public'],
     exclude=['ancestors_are_public'])
 
 
+class CategoryForm(BaseCategoryForm):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        if 'slug' in self.fields:
+            self.fields['slug'].required = False
+            self.fields['slug'].help_text = _('Leave blank to generate from category name')
```

Comment:
Is this string allready present in the gettext catalog?

Reply:
No - but I've been periodically updating the gettext catalog separately from functional changes, because it introduces a huge diff that makes review harder.